docs/frontend_utilisation_guide.md

```diff
@@ -24,48 +24,91 @@ The Pixel Streaming Frontend UI contains all the functionality for UI components
+
+### The Typescript Example
+The Typescript Example is a simple HTML, CSS and TypeScript implementation of what a user could create to initialize the Scalable Pixel Streaming Frontend Library. Its role is to instantiate the library's components and help start the Scalable Pixel Streaming connection to the signalling server. Please note  how a user chooses to initialize Library is open to interpretation and the users requirements and is not limited to the Typescript Example.
+## Getting Started; installation and consumption
+### Building for Development vs Production
+Within the Scalable Pixel Streaming Frontend there are several npm scripts that can be used to build the Library and Example for either development or production use cases. When building for development this will enable source maps for debugging. When building for production this will disable source maps, minimize console output and minimize the dist contents. Below are a list of npm scripts for both the Library and Example and what each command does:
+
+#### Library scripts
+Please ensure that all Library scripts are executed from the `library` directory and a user must first run `npm install` to install all the Libraries dependencies.  
+- `npm run build-dev`: Build the Library in development mode
+- `npm run build-prod`: Build the Library in production mode
+
+#### Example scripts
+Please ensure that all example scripts are executed from the `examples/typescript` directory. It is always recommended that a user installs the Library first as the example requires the Library. Contrary to this however, `build-all-dev` and `build-all-prod` do not require the Library to be installed and built first and will install and build both the Library and Example and all dependencies.
+- `npm run build-dev`: Build the Library in development mode
+- `npm run build-prod`: Build the Library in production mode
+- `npm run serve-dev`: Serve the Example locally using the Library in development mode
+- `npm run serve-prod`: Serve the Example locally using the Library in production mode
+- `npm run symlink`: Links the Library to the Example for consumption
+- `npm run build-all-dev`: Build the Library and the Example in development mode and link the Library to the Example for consumption
+- `npm run build-all-prod`: Build the Library and the Example in production mode and link the Library to the Example for consumption
+
+### Installing the Scalable Pixel Streaming Frontend Library From Github
```

Comment:
remove capitalisation of *From*

Reply:
resolved in b015fe9e45be6e7fe7028ce4bbd3d972cade4fdf